ulib/Makefile

```diff
@@ -6,27 +6,23 @@ FSTAR_HOME=..
 # Makefile that is distributed in the binary package. This
 # Makefile here is not.
 
-all:
+all: .cache
```

Comment:
`.cache` should be an order-only dependency, i.e. `all: | .cache` to avoid rebuilding the `.cache` target every time a file is added. Same below.

Reply:
Hi Santiago, Thanks for noticing this and commenting. Actually, maybe this dependence and the target for making .cache can be skipped altogether, since .cache is actually part of the repository (it contains the hints files that are checked in).